MTM-56104 add info about details of device counting for usage statistics (#2054)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-undefined-device-count-in-usage-statistics-detalis-explained.md

---
date: 2024-05-15
title: Device count details for usage statistics documented

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-56104
version: 10.20.0.0

---

> The details of the algorithms counting root devices, all devices, and leaf devices are explained in the documentation of Usage statistics and billing. 

Reviewer: ```suggestion
Details about the algorithms counting root devices, all devices, and leaf devices have been added to the user documentation, see [Usage statistics and billing](/enterprise-tenant/usage-and-billing/). 
```
